### PR TITLE
Sphinx: PDF, Link rst/md, png

### PR DESCRIPTION
**Pull request by ax3l**
- fix the pdf build
- link to markdown and rst resources
- link png for install.rst to `/`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,16 +144,13 @@ Before you post a question, browse the PIConGPU
...
 
-In addition, please consider using the collaborative features of GitHub if you
-have questions or comments on code or documentation. This will allow other
-users to see the piece of code or documentation you are referring to.
+In addition, please consider using the collaborative features of GitHub if you have questions or comments on code or documentation.
+This will allow other users to see the piece of code or documentation you are referring to.
 
-Main ressources are in our [online manual](https://picongpu.readthedocs.io), the [user section](https://github.com/ComputationalRadiationPhysics/picongpu/wiki) of our wiki, documentation files in `.md` and `.rst` format in this repository and a [getting started video](http://www.youtube.com/watch?v=7ybsD8G4Rsk).
+Main ressources are in our [online manual](https://picongpu.readthedocs.io), the [user section](https://github.com/ComputationalRadiationPhysics/picongpu/wiki) of our wiki, documentation files in [`.md` (Markdown)](http://commonmark.org/help/) and [`.rst` (reStructuredText)](http://www.sphinx-doc.org/en/stable/rest.html>) format in this repository and a [getting started video](http://www.youtube.com/watch?v=7ybsD8G4Rsk).
```
Comment (PrometheusPi) [inline README.md]: REQUIRED:
remove `>` from the reStructuredText URL

File: docs/source/install/path.rst [hunk 2]
```diff
@@ -8,63 +8,110 @@ Installation
...
+
+You choose a supported C++ compiler and configure, compile and install all missing dependencies from source.
+You are responsible to manage the right versions and configurations.
+Performance can be near-ideal if architecture is choosen correctly (and/or if build directly on your hardware).
+You then set environment variables to find those installs.
+
+Spack
+^^^^^
+
+[Spack]_ is a flexible package manager for HPC that can organize versions and dependencies for you.
```
Comment (PrometheusPi) [inline docs/source/install/path.rst]: OPTIONAL:
add HPC *systems*

File: examples/Bremsstrahlung/README.rst [hunk 3]
```diff
@@ -33,7 +33,7 @@ References
 .. [Jackson]
     J.D. Jackson.
     *Electrodynamics*,
-    Wiley‐VCH Verlag GmbH & Co. KGaA (1975),
+    Wiley-VCH Verlag GmbH & Co. KGaA (1975),
```
Comment (PrometheusPi) [inline examples/Bremsstrahlung/README.rst]: QUESTION:
What did you change with the `-` here?

Comment (ax3l) [inline examples/Bremsstrahlung/README.rst]: that was a weird, non utf8 character that broke latex

**Pull request merged**